packages/ui/src/services/colors-custom-classes/plugin/index.ts

```diff
@@ -0,0 +1,63 @@
+import { watch } from 'vue'
+
+import { isServer } from '../../../utils/ssr'
+import { useGlobalConfig } from '../../global-config'
+import { defineVuesticPlugin } from '../../vue-plugin/utils'
+import { addOrUpdateStyleElement } from '../../../utils/dom'
+
+import type { ColorVariables } from '../../color'
+import type { ColorsCustomClassesConfig, ColorsCustomClassesConfigItem } from '../types'
+
+import { ColorsCustomClassesPresets } from '../config/default'
+
+export const getColorsCustomClassesDefaultConfig = () => ColorsCustomClassesPresets
+
+const getColorsCustomClassesHelpers = (helpers: ColorsCustomClassesConfig, colors: ColorVariables) => {
+  const colorsEntries = Object.entries(colors)
+
+  return helpers.reduce((acc, helper: ColorsCustomClassesConfigItem) => acc.concat(
+    colorsEntries.map(([colorName, colorValue]) => ({
+      ...helper,
+      postfix: helper.postfix ? helper.postfix : colorName,
+      value: helper.value ? helper.value : colorValue,
+    })),
+  ), [] as ColorsCustomClassesConfig)
+}
+
+const getColorsCustomClassesStyles = (helpers: ColorsCustomClassesConfig) =>
+  helpers.reduce((styles, helper) => {
+    const style = [helper.property].flat().map(prop => `${prop}: ${helper.value};`).join('')
+
+    styles += `.va-${helper.prefix}--${helper.postfix} { ${style} }`
+
+    return styles
+  }, '')
```

Comment:
This is not one-liner, so prefer to {} or even function keyword. Can damage brain.
```suggestion
const getColorsCustomClassesStyles = (helpers: ColorsCustomClassesConfig) => {
  return helpers.reduce((styles, helper) => {
      const style = [helper.property].flat().map(prop => `${prop}: ${helper.value};`).join('')
  
      styles += `.va-${helper.prefix}--${helper.postfix} { ${style} }`
  
      return styles
    }, '')
  }
```